pkg/cluster/config/defaults/image.go

```diff
@@ -18,4 +18,4 @@ limitations under the License.
 package defaults
 
 // Image is the default for the Config.Image field, aka the default node image.
-const Image = "kindest/node:v1.14.1"
+const Image = "kindest/node:alpha-v1.14.1@sha256:1fb6e8a3842eed2ba0cb202ae1a70d208ed372c76b4600aa717c77d589f8334a"
```

Comment:
need to think more about naming, this is to signal that it requires an unreleased kind version to run, since we introduced the default manifest templating. might need to rethink that mechanism too.

/hold
 
also after some more though, we should be pinning these by sha always.

Reply:
Kubernetes 1.14.2 is cut tomorrow. We can put kind 0.3 on this and document the image breakage in the release notes as well as document and encourage pinning image shas.